tox.ini

```diff
@@ -30,26 +33,27 @@ set_env =
 pass_env =
     CRAFT_PROVIDERS_TESTS_*
     CI
+    PYTEST_ADDOPTS
```

Comment:
Funny that none of us noticed (at least I didn't) ` PYTEST_ADDOPTS: "--no-header -vv -rN"` was not getting used in the github workflows

Reply:
![image](https://github.com/canonical/craft-providers/assets/4305943/c013cecb-29d8-45fc-b81e-3e02ed8a89b3)